unicorn/include/cocaine/api/v15/unicorn.hpp

```diff
@@ -15,6 +16,22 @@ class unicorn_t: public api::unicorn_t {
     virtual
     unicorn_scope_ptr
     named_lock(callback::lock callback, const unicorn::path_t& path, const unicorn::value_t& value) = 0;
+
+    virtual
+    unicorn_scope_ptr
+    create_with(callback::create callback, const unicorn::path_t& path, const unicorn::value_t& value, const cocaine::dynamic_t& args) {
```

Comment:
looks like this one should not be the part of unicorn API, instead it is a part of unicorn service API